bot.py

```diff
@@ -9,24 +9,31 @@ async def on_ready():
   print(f"We are live as {client.user}")
 
 @client.event
-async def on_message(message):
-  if "twitch()" == message.content.lower() and 'commands' == str(message.channel):
-    await message.channel.send("Go Follow Melkeydev over at https://www.twitch.tv/melkeydev")
+async def on_message(msg):
+  channelName = str(msg.channel)
 
-  if "test()" == message.content.lower() and 'commands' == str(message.channel):
-    await message.channel.send("Test worked you didnt break it yet")
+  if channelName != "commands" or channelName != "bot-construction":
+    return
 
-  if "schedule()" == message.content.lower() and 'commands' == str(message.channel):
-    await message.channel.send("Melkey streams start on Mondays, Wednesdays, and Fridays at 9PM EST")
+  cmd = msg.content.lower()
 
-  if "project()" == message.content.lower() and 'commands' == str(message.channel):
-    await message.channel.send("Melkey is working on a NBA app written in react to search, and compare player stats! You can find it over at https://github.com/Amokstakov/NewNBAApp")
+  if cmd == "twitch()":
+    await msg.channel.send("Go Follow Melkeydev over at https://www.twitch.tv/melkeydev")
 
-  if "pow()" == message.content.lower() and 'commands' == str(message.channel):
-    await message.channel.send("The pow is a sacred technique practice by the ancient tribes of Konoha.")
+  if cmd == "test()":
+    await msg.channel.send("Test worked you didnt break it yet")
 
-  if "crash()" == message.content.lower() and 'commands' == str(message.channel):
-    await message.channel.send("Nice try I am impossible to crash")
+  if cmd == "schedule()":
+    await msg.channel.send("Melkey streams start on Mondays, Wednesdays, and Fridays at 9PM EST")
+
+  if cmd == "project()":
+    await msg.channel.send("Melkey is working on a NBA app written in react to search, and compare player stats! You can find it over at https://github.com/Amokstakov/NewNBAApp")
```

Comment:
we need to update this due to the repo going private

Reply:
I'll remove the link for now and then in the future, replace it with the link to the production website instead.